controller/TaskQueueData.php

```diff
@@ -37,15 +36,14 @@
  */
 class TaskQueueData extends AbstractBookletController
 {
-
     /**
      * Lists all tasks related to booklet create/regenerate
      */
     public function getTasks()
     {
-        $user = common_session_SessionManager::getSession()->getUser();
+        $user = $this->getSession()->getUser();
```

Comment:
Missing call for `$this->defaultData()`
also for public methods `getStatus` , `archiveTask`, `downloadTask`